pkg/common/parameters.go

```diff
@@ -30,6 +30,7 @@ const (
 	ParameterKeyProvisionedIOPSOnCreate       = "provisioned-iops-on-create"
 	ParameterKeyProvisionedThroughputOnCreate = "provisioned-throughput-on-create"
 	ParameterAvailabilityClass                = "availability-class"
+	ParameterKeyEnableConfidentialCompute     = "enable-confidential-storage"
```

Comment:
This param needs to be called enable-confidential-storage? The naming should not be changed publicly but we can convert it within the driver to EnableConfidentialCompute.